include/boost/geometry/strategy/cartesian/precise_area.hpp

```diff
@@ -0,0 +1,118 @@
+// Boost.Geometry
+
+// Copyright (c) 2020, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
+#ifndef BOOST_GEOMETRY_STRATEGY_CARTESIAN_AREA_ACCURATE_HPP
```

Comment:
Macro name does not correspond to file name.

Reply:
OK this is from PR https://github.com/boostorg/geometry/pull/749 but I am fixing it here.